src/compiler/transformers/transform-utils.ts

```diff
@@ -165,6 +165,12 @@ export const createStaticGetter = (propName: string, returnExpression: ts.Expres
   );
 };
 
+/**
+ * Retrieves a value represented by TypeScript's syntax tree by name of a static getter. The value is transformed to a TypeScript value.
```

Comment:
Should we possibly call a "TypeScript Value" a "runtime value" here, for less ambiguity?

Reply:
Good call - 5ebde1635